LOCALIZATION.md

```diff
@@ -3,25 +3,51 @@
 
+Contributions to improving the following localization guides and localization policies are also welcome.
+
+---
+
+# Initiating a new localization team
+
+## Basic requirements to initiate a new localization team
+
+- The Glossary project does not have a localization team for your language yet
+  - check the language list from [localization progress](https://github.com/cncf/glossary/projects/2) and ask in the [glossary-localizations channel in Slack](https://cloud-native.slack.com/archives/C02N2RGFXDF). There may be a group of volunteers who haven't gotten started yet.
 - At least three volunteers
 - At least two approvers 
-  - Each term must be approved by two approvers
-  - Unanimous consent must be reached to be approved 
+  - two approvers must approve each term
+  - unanimous consent must be reached to be approved 
+  - at least one volunteer must have basic GitHub knowledge and be willing to help the others get up to speed (such as PR, review process, branch, ..)
 
 ## Instructions to initiate a new localization
 
-### Open an Issue to notify initiation of a new localization 
+### 1. Opening an issue to notify initiation of a new localization 
+
+[Open an issue](https://github.com/cncf/glossary/issues/new/choose) 
+and select the `Initiate a New Localization Team` template.
+
+With this issue, new localization teams are requesting:
+ - their own development branch (ex: [dev-ko](https://github.com/cncf/glossary/tree/dev-ko) branch). 
+ - permission for approving localized content
+   ```
+   l10n team approvers will have push permission to this repository.
+   They will manage (merge) PRs for their l10n contributions in their development branch.
+   However, their ability to merge PRs to the `main` branch is restricted.
+   Please note: although they can review and approve PRs to the `main` branch, they should not approve PRs that aren't related to their own localization teams.
+   ``` 
 
-[Open an Issue](https://github.com/cncf/glossary/issues/new/choose) 
-based on the `Initiate a New Localization Team` Issue-template.
+Localization teams should use their assigned development branch for the following localization tasks (example of a dev branch: https://github.com/cncf/glossary/tree/dev-ko; dev-ko stands for Korean dev branch). 
 
-Through this issue, the new localization team will
- - get a permission for approving localization contents.
- - get a development branch for the localization works. (use the development branch for following localization tasks)
+### 2. Configuring a new language
 
-### Add a new language setting to site configuration
+Open a PR with the localization initiation following this example: https://github.com/cncf/glossary/pull/291.
+
+
+
+#### 2-1. Adding a new language setting to site configuration
 
 New languages need to be set up inside of `config.toml`.
 Find existing `[languages]` block in `config.toml` and add a block for the new language below other language blocks.
```

Comment:
```suggestion
To add a new language to site, you needs to modify [config.toml](https://github.com/cncf/glossary/blob/main/config.toml#L54) (Note: localization teams should use their assigned development branch for this task).
`[languages]` block of `config.toml` is used to configure languages for the site. `[languages.en]` means language configuration for English and `[languages.ko]` means language configuration for Korean.
Go to `[languages]` block in `config.toml` and add a new block like `[languages.ko]` with a language-specific configuration.
For instance, Korean localization team can add `[languages.ko]` block after `[languages.en]` block.
```
How about this ? :)

Reply:
Looks good. I made a few edits. Can you verify, it's all correct. If so, let's commit.